src/Loader.php

```diff
@@ -760,6 +762,21 @@ public static function get_order_statuses( $statuses ) {
 		return $formatted_statuses;
 	}
 
+	/**
+	 * Get all order statuses present in analytics tables that aren't registered.
+	 *
+	 * @return array Unregistered order statuses.
+	 */
+	public static function get_unregistered_order_statuses() {
+		$registered_statuses   = wc_get_order_statuses();
+		$all_synced_statuses   = OrdersDataStore::get_all_statuses();
+		$unregistered_statuses = array_diff( $all_synced_statuses, array_keys( $registered_statuses ) );
+		$formatted_status_keys = self::get_order_statuses( array_fill_keys( $unregistered_statuses, '' ) );
+		$formatted_statuses    = array_keys( $formatted_status_keys );
+
+		return array_combine( $formatted_statuses, $formatted_statuses );
```

Comment:
Are the readable strings not obtainable, ie "Partial payment"?

![Screen Shot 2020-04-10 at 10 50 20 AM](https://user-images.githubusercontent.com/1922453/78947346-2f870c80-7b19-11ea-8658-071afeea9110.png)

Reply:
Oh ok, I see the status name isn't available when the plugin is disabled.